conf/deployment/rosa/rosa_1az_4xl_1t_3w.yaml

```diff
@@ -0,0 +1,34 @@
+---
+ENV_DATA:
+  platform: 'rosa'
+  deployment_type: 'managed'
+  addon_name: 'ocs-converged'
+  configs: 'aws,stage'
+  worker_instance_type: 'm5.4xlarge'
+  worker_replicas: 3
+  master_replicas: 3
+  # size of storage cluster before expansion is 1 TiB
+  size: '1'
+
+# AUTH:
+#  openshiftdedicated:
+#    token: OCM TOKEN KEY PLACEHOLDER
+
+DEPLOYMENT:
+  force_download_ocm_cli: True
+  ocm_cli_version: '0.1.46'
```

Comment:
I saw that the version is different in this file https://github.com/red-hat-storage/ocs-ci/blob/master/conf/deployment/rosa/managed_3az_provider_qe_3m_3w_m54x.yaml. It's '0.1.60'.
Same thing about the `rosa_cli_version` - I see it's '1.1.7'